README.md

```diff
@@ -37,6 +37,9 @@ EB
 
 $ curl ifconfig.co/city
 Bornyasherk
+
+$ curl ifconfig.co/asn
+AS59795,Hosting4Real
```

Comment:
I'd prefer not to have formatting in the plain text responses, how about returning just the ASN? If clients want the org string, they can use `/json`.